### PR TITLE
Us 5 filtering job profile

### PR DESCRIPTION
**Pull request by msolecka-kainos**

### CODE REVIEW

File: src/main/resources/assets/js/application/recruiter/directives/tdprPersonDirective.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/rec
                     _init();
                 });
 
+                scope.$watch("person", function(){
```
Comment (mateuszc-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprPersonDirective.js:15]: There may be a better way to do this without $watch

File: src/main/resources/assets/js/application/recruiter/views/tdpr-directive-filter.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<form>
+    <label ng-repeat="job in jobProfiles">
+        <input type="radio" ng-model="jobProfile" name="jobProfile" ng-click="changeJobProfile($index)">
+        {{job}}
+    </label><br/>
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-directive-filter.html]: < br > elements are not really used anymore

File: src/main/resources/assets/js/application/recruiter/directives/tdprPersonDirective.js (hunk 1 above)
Comment (msolecka-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprPersonDirective.js:15]: We need this to refresh table.

Comment (mateuszc-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprPersonDirective.js:15]: Ok

File: src/main/resources/assets/js/application/recruiter/directives/tdprPersonsDirective.js [hunk 3]
```diff
@@ -50,13 +54,22 @@ define(['angular', 'application/recruiter/tdprRecruiterModule'], function (angul
...
+
                 scope.$watch('selectedPersonTable', function (newValue, oldValue) {
                     _refreshList(newValue);
                 });
 
                 scope.$parent.$watch('startWeekDay', function (newValue, oldValue) {
                     _init();
                 });
+
+                scope.$parent.$parent.$watch('jobProfile', function (newValue, oldValue) {
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprPersonsDirective.js:69]: woah, that's quite the genealogical tree you're trying to dig out

**Pull request merged**